blocks/api/test/parser.js

```diff
@@ -150,7 +150,7 @@ describe( 'block parser', () => {
 			} );
 
 			const parsed = parse(
-				'<!-- wp:core/test-block smoked:yes -->' +
+				'<!-- wp:core/test-block smoked="yes" url="http://google.com" -->' +
```

Comment:
What does this new `url` attribute add?  Let's have a test for spaces and some special characters like `'&` instead.